### PR TITLE
API documentation for Python get_stat_all()

### PR DESCRIPTION
**Pull request by shifucun**

### CODE REVIEW

File: api/python/stat_all.md [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+        "observationPeriod": "P1Y",
+        "importName": "CensusPEPSurvey",
+        "provenanceDomain": "census.gov"
+      }
+    ],
+  }
+}
+```
+
+In the next example, there is not data found so the API throws ValueError:
```
Comment (tjann) [inline api/python/stat_all.md:110]: s/not/no

File: api/python/stat_all.md [hunk 2]
```diff
@@ -0,0 +1,117 @@
+---
+layout: default
+title: Place Statistics - All
+nav_order: 14
+parent: Python
+grand_parent: API
+---
+
+# Get a Collection of Statistical Data for Multitple Places
```
Comment (tjann) [inline api/python/stat_all.md:9]: s/Multitple/Multiple

File: api/python/stat_all.md [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+**Arguments**
+
+- `places (Iterable of str)`: The `dcid`s of the [`Place`](https://datacommons.org/browser/Place) to query for.
+
+- `stats_vars (Iterable of str)`: The `dcid`s of the
+  [`StatisticalVariable`](https://datacommons.org/browser/StatisticalVariable).
+
+**Returns**
+
+A nested `dict` mapping Places to StatisticalVariables and all available
```
Comment (tjann) [inline api/python/stat_all.md:24]: Add backticks to `Place`s and `StatisticalVariable`s to keep things consistent in this page?

File: api/python/stat_all.md [hunk 4]
```diff
@@ -0,0 +1,117 @@
...
+time series for each Place and StatisticalVariable pair.
+
+The top level `dict` key is the place dcid and the second level `dict` key is the
```
Comment (tjann) [inline api/python/stat_all.md:27]: s/place/`Place`
s/statistical variable/`StatisticalVariable`

File: api/python/stat_all.md [hunk 5]
```diff
@@ -0,0 +1,117 @@
...
+statistical variable dcid, with the object being an array of time series object
+with the following fields
+
+- `val`: an object from date to statistical value.
```
Comment (tjann) [inline api/python/stat_all.md:31]: s/object/`dict` ?

File: api/python/stat_all.md [hunk 6]
```diff
@@ -0,0 +1,117 @@
...
+- `importName`: the import name of the observations.
+- `provenanceDomain`: the provenance domain of the observations.
```
Comment (tjann) [inline api/python/stat_all.md:33]: s/provenance/[`Provenance`](https://datacommons.org/browser/Provenance)

Comment (tjann) [reply api/python/stat_all.md:33]: same with measurement method, observation period, unit, and scalingFactor

File: api/python/stat_all.md [hunk 7]
```diff
@@ -0,0 +1,117 @@
...
+- `measurementMethod`: the measurement method of the observations, if it exists.
+- `observationPeriod`: the observation period of the observations, if it exists.
+- `unit`: the unit of the observations, if it exists.
+- `scalingFactor`: the scaling factor of the observations, if it exists.
+
+**Raises**
+
+- `ValueError` - If no statistical value found for the place with the given parameters.
```
Comment (tjann) [inline api/python/stat_all.md:41]: This is for multiple statvar and multiple places-- If no statistical value found for any `Place` and `StatisticalVariable` combinations?

Or, more general:

"* `ValueError` - If no statistical values found for the given parameters."

File: api/python/stat_all.md [hunk 8]
```diff
@@ -0,0 +1,117 @@
...
+
+Be sure to initialize the library. Check the [Python library setup guide](/api/python/) for more details.
+
+You can find a list of StatisticalVariables with human-readable names [here](/statistical_variables.html).
```
Comment (tjann) [inline api/python/stat_all.md:45]: s/StatisticalVariables/`StatisticalVariable`

File: api/python/stat_all.md (hunk 1 above)
Comment (shifucun) [reply api/python/stat_all.md]: done

File: api/python/stat_all.md (hunk 2 above)
Comment (shifucun) [reply api/python/stat_all.md]: done

File: api/python/stat_all.md (hunk 3 above)
Comment (shifucun) [reply api/python/stat_all.md]: done

File: api/python/stat_all.md (hunk 4 above)
Comment (shifucun) [reply api/python/stat_all.md]: done

File: api/python/stat_all.md (hunk 5 above)
Comment (shifucun) [reply api/python/stat_all.md]: done

File: api/python/stat_all.md (hunk 6 above)
Comment (shifucun) [reply api/python/stat_all.md]: done

**Comment by tjann** [comment]
LG, thanks!

**Comment by tjann** [comment]
Not really related to this PR, but do you think we should unlist the get_stats func for python and REST? Now that we have all of the new ones.

**Comment by shifucun** [comment]
> Not really related to this PR, but do you think we should unlist the get_stats func for python and REST? Now that we have all of the new ones.

Good idea. Can remove in the follow up.

**Pull request merged**